packages/db/src/runtime/db-client.ts

```diff
@@ -71,7 +71,23 @@ export function createRemoteDatabaseClient(appToken: string, remoteDbURL: string
 				});
 			}
 
-			if (method === 'run') return remoteResult;
+			if (method === 'run') {
+				const rawRows = Array.from(remoteResult.rows);
+				// in order to serialize properly much implement toJSON
```

Comment:
```suggestion
				// Implement basic `toJSON()` for Drizzle to serialize properly
```

Cleaning up this comment a little. Am I right this the reason for `toJSON()`? For Drizzle to call internally?

Reply:
Yep, thanks!